src/rpc_mock.rs

```diff
@@ -0,0 +1,155 @@
+                    }
+                    MockRpcRequest::GetTransactionCount => {
+                        Ok(Value::Number(Number::from(1234)))
+                    }
+                    MockRpcRequest::SendTransaction => {
+                        Ok(Value::String(
+                            "43yNSFC6fYTuPgTNFFhF4axw7AfWxB2BPdurme8yrsWEYwm8299xh8n6TAHjGymiSub1XtyxTNyd9GBfY2hxoBw8".to_string(),
+                        ))
+                    }
+                    _ => {
+                        Ok(Value::Null)
+                    }
+                }
+            }
+            "airdrop" => {
+                match self {
+                    MockRpcRequest::GetBalance => {
+                        Ok(Value::Number(Number::from(0)))
+                    }
+                    MockRpcRequest::GetLastId => {
+                        Ok(Value::String(
+                            "7RoSF9fUmdphVCpabEoefH81WwrW7orsWonXWqTXkKV8".to_string(),
+                        ))
+                    }
+                    MockRpcRequest::GetSignatureStatus => {
+                        Ok(Value::String(
+                            "Confirmed".to_string(),
+                        ))
+                    }
+                    MockRpcRequest::SendTransaction => {
+                        Ok(Value::String(
+                            "43yNSFC6fYTuPgTNFFhF4axw7AfWxB2BPdurme8yrsWEYwm8299xh8n6TAHjGymiSub1XtyxTNyd9GBfY2hxoBw8".to_string(),
+                        ))
+                    }
+                    _ => {
+                        Ok(Value::Null)
+                    }
+                }
+            }
+            "bad_sig_status" => {
+                Ok(Value::String("Nonexistant".to_string()))
+            }
+            "account_in_use" => {
+                match self {
+                    MockRpcRequest::GetLastId => {
+                        Ok(Value::String(
+                            "7RoSF9fUmdphVCpabEoefH81WwrW7orsWonXWqTXkKV8".to_string(),
+                        ))
+                    }
+                    MockRpcRequest::GetSignatureStatus => {
+                        Ok(Value::String(
+                            "AccountInUse".to_string(),
+                        ))
+                    }
+                    MockRpcRequest::SendTransaction => {
+                        Ok(Value::String(
+                            "43yNSFC6fYTuPgTNFFhF4axw7AfWxB2BPdurme8yrsWEYwm8299xh8n6TAHjGymiSub1XtyxTNyd9GBfY2hxoBw8".to_string(),
+                        ))
+                    }
+                    _ => {
+                        Ok(Value::Null)
+                    }
+                }
+            }
+            _ => {
+                Ok(Value::Null)
+            }
+        }
+    }
+}
+
+pub fn request_airdrop_transaction(
+    _drone_addr: &SocketAddr,
+    _id: &Pubkey,
+    tokens: u64,
+    _last_id: Hash,
+) -> Result<Transaction, Error> {
+    if tokens == 0 {
+        Err(Error::new(ErrorKind::Other, "Airdrop failed"))
```

Comment:
Generally, we want to do early returns in code like this so that you don't need that `else` and the code stays nice and flat. Besides just being easier to read, validation checks come and go over time, so it's nice when the indentation doesn't change each time.